packages/generator-electrode/component/templates/packages/component/lintstagedrc

```diff
@@ -0,0 +1,6 @@
+{
+  "{archetype,config,src,test}/**/*.{js,jsx}": [
```

Comment:
@didi0613 fixed in https://github.com/electrode-io/electrode/pull/351/commits/39d4a89df5507d9af1d6829192e87f401996c614